src/sage/libs/singular/polynomial.pyx

```diff
@@ -267,6 +268,15 @@ cdef int singular_polynomial_cmp(poly *p, poly *q, ring *r) noexcept:
         sage: P(2) > P(1)
         True
 
+    ::
+
+        sage: R.<x, y> = Integers(10)[]
+        sage: l = [i*x+j*y+k for i in range(10) for j in range(10) for k in range(10)]
```

Comment:
```
R.<x,y> = Integers(10)[]
```
no space after the comma, which is the usual style in Sage.

Reply:
```
[sage]$ rg '\.<\w,\w>' |wc -l
4151
[sage]$ rg '\.<\w, \w>' |wc -l
470
```

Looks like all over the place, but sure.